client/src/components/ApplicantLogin.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react";
+import { Button, Form } from "semantic-ui-react";
+import { Redirect } from "react-router-dom";
+import { signApi } from "../api/auth";
+
+const ApplicantLogin = () => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [error, setError] = useState(null);
+  const [success, setSuccess] = useState(false);
+
+  const handleChange = event => {
+    if (event.target.name === "email") {
+      setEmail(event.target.value);
+    }
+    if (event.target.name === "password") {
+      setPassword(event.target.value);
+    }
+  };
+
+  const loginSubmit = () => {
+    signApi(email, password)
+      .then(data => {
+        const token = data.token;
+        console.log(token);
+
+        localStorage.setItem("token", token);
+        localStorage.setItem("user", JSON.stringify(data.user));
+        // document.location.reload();
+        setSuccess(true);
+      })
+      .catch(() => {
+        setError(true);
+        setSuccess(false);
+      });
+  };
+  console.log(success);
```

Comment:
console.log should be removed.

Reply:
Thanks for the review. I've fixed the code. Could you review it again, please? Thanks.